FirstTask.py

```diff
@@ -0,0 +1,201 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, name, password, account_no, balance, account_type):
+        self.name = name
+        self.balance = balance
+        self.account_no = account_no
+        self.account_type = account_type
+        self.password = password
+
+    def account_number(self):
+        return self.account_no
+
+    def pass_code(self):
+        return self.password
+    
+    def check_balance(self):
+        return self.balance
+
+    def check_account_type(self):
+        return self.account_type
+    
+    def deposit_bill(self, bill):
+        if bill >= 0:
+            self.balance += bill
+            return True
+        else:
+            return False
+    
+    def draw_money(self, money):
+        if self.balance >= money:
+            self.balance -= money
+            return True
+        else:
+            return False
+    
+    def account_details(self):
+        print("Account # ", self.account_no)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+        print("Account Type: ", self.account_type)
+    
+class Bank():
+    @staticmethod
+    def login(account_list, account_number, code):
+        sys('clear')
+        check1 = False
+        for i in range(len(account_list)):
+            if account_number == account_list[i].account_number() and code == account_list[i].pass_code():
+                return True,account_list[i]
+        return False
+    
+    @staticmethod
+    def transfer_money(sender_account_number, receiver_account_number, amount):
+        if receiver_account_number.deposit_bill(amount):
+            sender_account_number.balance -= amount 
+            return True
+
+    @staticmethod
+    def compare(current_account, account_list, other_account_number):
+        check2 = False
+        for j in range(len(account_list)):
+            if other_account_number == account_list[j].account_number():
+                check2 = True
+                print("\nYour Details: ")
+                BankAccount.account_details(current_account)
+                print("\nOther Account's Details: ")
+                BankAccount.account_details(account_list[j])
+                if current_account.check_balance() > account_list[j].check_balance():
+                    return 1
+                elif current_account.check_balance() == account_list[j].check_balance():
+                    return 0
+                else:
+                    return -1
+        if not check2:
+            print("\nAccount Number Doesn't Exist\n\n")
+
+    @staticmethod
+    def find_account(account_number):
+        account_number.account_details()

```

Comment:
This method should find the account object from the `account_list` which should be an attribute of this `Bank` class.